onnx/onnx.proto

```diff
@@ -103,6 +114,7 @@ message GraphProto {
   repeated TensorProto initializer = 5;
 
   // The version of the IR this graph targets. See Version enum below.
+  // This field MUST be present this version of the IR.
   optional int64 ir_version = 6;
```

Comment:
we should probably bump IR version because it's indeed a breaking change (just following the protocol)

Reply:
Yup. See my last comment.

Reply:
Will do.